packages/rocket-fileuploader-aws-infrastructure/README.md

```diff
@@ -0,0 +1,67 @@
+# File Uploader Booster Rocket for AWS
+
+This package is a configurable Booster rocket to add an Object Storage(S3 bucket) to your Booster Application. 
+Every time a new object is uploaded, a new event will be persisted in the Events Store will data like the fileUri and file size.
+
+## Usage
+
+Install this package as a dependency in your Booster project.
+
+```sh
+npm install @boostercloud/rocket-static-sites-aws-infrastructure
```

Comment:
```suggestion
npm install --save-dev @boostercloud/rocket-static-sites-aws-infrastructure
```

Reply:
Synced offline, we need it as dependency because of lambda code

Reply:
Could you please tell the rest why? From my point of view, the lambda code gets deployed on deployment time, so we should not need the rocket code in runtime anymore ❔